src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+import { User } from '../../types/Users';
+
+export const UserInfo: React.FC<{ user: User }> = ({ user }) => (
```

Comment:
Destructure the user, please. And create a type for Props
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => (
```